pkg/service/array_service.go

```diff
@@ -106,3 +106,48 @@ func (svc *ArrayService) DeleteArray(id string) error {
 	}
 	return nil
 }
+
+// FailoverArray - perform a failover on the specified array.
+//   Required parameters:
+//       id - ID of the array to perform failover on.
+
+//   Optional parameters:
+//       force - Initiate failover without performing any precheck.
+
+func (svc *ArrayService) FailoverArray(id string, force *bool) error {
+
+	if len(id) == 0 {
+		return fmt.Errorf("error: invalid parameter specified id:%v, force:%v ", id, force)
```

Comment:
The intention was to log all the arguments for the end-user . we could remove it and display the only argument under validation.

Reply:
Since this is generated code, I realize exceptions to the rule are annoying.  But, I think customers would find logging of passwords to be objectionable (or worse).  Maybe the generator could skip validation of parameters named "password"?

Reply:
I agree. will remove 'password' from logging.

Reply:
Thanks.  Since this message is only generated when an argument is invalid, its purpose seems to be to help identify the invalid argument.  Including non-validated parameters might be more confusing than helpful.  So, I agree with your idea to include only the arguments under validation.

Reply:
It was getting complicated to handle individual 'password' argument from the line so i kept it and now printing only the address of it . That should be fine . So, it will actually log the address of password not the actual password .

Reply:
I find printing the address of the password (or anything for that matter) useless.  Why can't we just skip that?

Reply:
Yes, i agree 
Freemarker tool doesn't not have a good programming language syntax even doing a simple thing is complex, the generator process the Action for all the Objects and their Arguments. It can be implemented by selecting the argument 'password' and building logic to ignore it. But it is not worth the effort I feel, do we still need it?

Reply:
Yes, I feel we should.  We actually do something similar in the java SDK where we look for the keywords password, token, session, etc. and make sure to blank those out.

Reply:
Sure , i will address this issue via a separate bug. thanks